### PR TITLE
feat: expose lc39 both as cli and as function

### LINKED ISSUE #212: Expose lc39 function to start the server
## The feature or bug you are proposing
Instead of use lc39 only as a command, expose also an lc39 function to init the server (such as for the exported test function)

## The description of the bug or the rationale of your proposal
Using it in the Dockerfile using node and tini, the command is accessed using `./node_modules/.bin/lc39`, it would be better to simplify it.

### Your environment

lc39: 6.0.1

### PR DESCRIPTION
**Pull request by davidebianchi**
This PR contains breaking changes, so it is based to branch with upgrade to fastify-v4.

Resolves #212 

#### Checklist

- [x] your branch will not cause merge conflict with `master`
- [x] run `npm test`
- [x] tests are included
- [x] the documentation is updated or integrated accordingly with your changes
- [x] the code will follow the lint rules and style of the project
- [x] you are not committing extraneous files or sensible data

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -5,9 +5,12 @@ name: Node.js CI
...
 on:
   push:
-    branches: [ master ]
+    branches:
+      - master
   pull_request:
-    branches: [ master ]
+    branches:
+      - master
+      - fastify-v4
```
Comment (fredmaggiowski) [inline .github/workflows/node.js.yml:13]: can this be removed or is still useful?

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,12 +12,40 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Changes
 
+- exported lc39 function remove the default log level set to silent, and it is now set to info. This is required since it is possible to configure lc39 also from function and not only from CLI
+### Changes
```
Comment (fredmaggiowski) [inline CHANGELOG.md:16]:
```suggestion
```

File: tests/modules/module-with-transform-schema.js [hunk 3]
```diff
@@ -15,10 +15,21 @@ module.exports = async function plugin(fastify) {
...
+        hide: true,
+      },
+    }
+  }
   if (schema.hide) {
     // if a route is set as hidden its schema should not be modified
-    return schema
+    return {
+      schema,
+      url,
```
Comment (fredmaggiowski) [inline tests/modules/module-with-transform-schema.js:31]: why adding the url now? 🤔

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (davidebianchi) [reply .github/workflows/node.js.yml:13]: It is useful to run tests, I will remove it before merge the fasting-v4 branch in master

File: tests/modules/module-with-transform-schema.js (hunk 3 above)
Comment (davidebianchi) [reply tests/modules/module-with-transform-schema.js:31]: With fastify swagger at v4, the transform function is changed and it requires to return an object with schema and url. It's a bug on a test of the fastify-v4 branch that I discovered with this changes

**Comment by davidebianchi** [comment]
I should update of the typings before merge this PR

**Comment by davidebianchi** [comment]
Done, it is now ready to the review

**Review by fredmaggiowski — APPROVED** [review]

**Pull request merged**